lib/lvqueue.bpf.c

```diff
@@ -86,7 +86,7 @@ int lvq_push(lv_queue_t __arg_arena *lvq, u64 val)
 		lvq->cur = newarr;
 	}
 
-	lv_arr_put(arr, b, val);
+	lv_arr_put(newarr, b, val);
```

Comment:
This should be `lv_arr_put(lv->cur, b, val)` because newarr is only initialized inside the if block.